Vizzuality/heco-invest
# Fix/transfer ownership of account invitations

Transferring ownership of invitations along with account ownership.

## Testing instructions

Invite user to account, change ownership, check if invited_by_id changed to new owner id

## Tracking

https://vizzuality.atlassian.net/browse/LET-849
